content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Automate the Remediation of Detected Threats with Webhooks
+kind: guide
+further_reading:
+- link: "/security_platform/explorer/"
+  tag: "Documentation"
+  text: "Start investigating signals in the Signals Explorer"
+
+---
+
+## Overview
+
+[Security Monitoring][1] allows you to set Detection Rules that trigger auto-remediation workflows. With Datadog’s [webhook integration][2], set up webhooks to deliver payloads to the services you want to automate whenever a [Detection Rule][3] is broken. Every webhook payload contains information about the triggering event and a custom message that can be used to initiate services downstream. Automate commands for any service that has a webhook URL.
```

Comment:
> is broken

this should say something a bout when a detection rule is triggered. not when something is broken.

> be used to initiate services downstream

I wouldn't say "initiate". I would say something like "can be passed to services downstream" or "can be used by downstream services".

Reply:
+1 on adding more detail, any suggestions? Following what the blog says, and not quite sure what we mean by commands here -- actual commands or the automated processes that occur within a service?

Reply:
@jmassey-datadog Bumping for further review 😸

Reply:
these "commands" are actually talking about the scenarios below. So maybe a single sentence summarizing the 3 different scenarios?  Or something like

> Security Orchestration and Automation Response tools accept incoming HTTP requests and these webhooks will initiate any workflow you have defined.